docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -145,16 +155,18 @@ The following example shows how to create a gallery-like flexbox layout using th
     nextButton.Label = labelMap2;
     ```
 
-8.  Add the image to the content area:
+9.  Add the image to the content area:
 
-    ```
+    ```csharp
     /// Add an image to the center of the content area
-    ImageView imageView = new ImageView("image.jpg");
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "image.png");
     imageView.ParentOrigin = ParentOrigin.TopLeft;
     imageView.PivotPoint = PivotPoint.TopLeft;
     content.Add(imageView);
     ```
 
+For more information, see [the full example](download link)
+
 <a name="properties"></a>
 ## Modifying FlexContainer Properties
```

Comment:
Global: 
Can we please use sentence case for headings?

Reply:
In other guides headings looks like this one. Are you sure that we should apply this?